### ISSUE TITLE
Cloudscale volume module still using server_uuids

### ISSUE THREAD

**Issue by id101010** [cloudscale-ch/ansible-collection-cloudscale#53]
While debugging an issue with our cloudscale volumes I've noticed that the module still makes use of `server_uuids` despite the api documentation mentioning it being deprecated and recommending the use of `servers` instead. Could this be an issue?

See: 
* https://github.com/cloudscale-ch/ansible-collection-cloudscale/blob/master/plugins/modules/volume.py#L207
* https://www.cloudscale.ch/en/api/v1#volumes

**Comment by href** [comment]
Thanks for bringing this to our attention. While we certainly should support  `servers`, there's nothing inherently wrong with `server_uuids` and it should work as is for the time being.

I have created an internal ticket to make sure we support `servers` in the near future, but the use of `server_uuids` should not cause any issue in the meantime (I just tested it locally and it worked as expected).

We'll update this ticket once this has been implemented.
